.env.sample

```diff
@@ -29,8 +29,17 @@ TCHAP_LOG_SENSITIVE_DATA=true
 TCHAP_OTP_MAIL_DELAY_IN_MINUTES=0
 TCHAP_CODE_TIMEOUT_IN_MINUTES=60
 
+TCHAP_BOT_ACCOUNT_EMAIL=
+TCHAP_BOT_PASSWORD=""
+#TCHAP_BOT_MATRIX_ID= not used
+
+# feature flags
+FEATURE_TCHAP_BOT_OTP=false
+
```

Comment:
verifier si il faut changer les actions pour la livraison (TCHAP_BOT_MATRIX_ID=not used) : https://github.com/tchapgouv/tchap-identite/issues/113